utils/eventListeners.ts

```diff
@@ -100,6 +100,15 @@ export function filterEventData(
     .map((event) => event.event.toHuman().data);
 }
 
+export function filterZeroEventData(
```

Comment:
This method filters the event collection by the method param. 
So what does a name like "filterEventsByMethodName" look like?

+ Pz follow linter rules. I guess that this chunk of code does not match the linter rules, 
```


) {
  const filteredResult = result.filter((x) => x.method === method);
  if (filteredResult[0] === undefined){return undefined}else{
  return (JSON.parse(JSON.stringify(filteredResult[0].event.toHuman().data)));}
}
```

Reply:
Fixed